packages/env/lib/build-docker-compose-config.js

```diff
@@ -176,26 +176,41 @@ module.exports = function buildDockerComposeConfig( config ) {
 				image: 'mariadb',
 				ports: [ '3306' ],
 				environment: {
-					MYSQL_ALLOW_EMPTY_PASSWORD: 'yes',
+					MYSQL_ROOT_PASSWORD: 'password',
+					MYSQL_DATABASE: 'wordpress',
 				},
 				volumes: [ 'mysql:/var/lib/mysql' ],
 			},
+			'tests-mysql': {
+				image: 'mariadb',
+				ports: [ '3306' ],
```

Comment:
This port is already being used by the `mysql` container above.

Reply:
I think those ports just denote which ports are exposed within a container. Ports are isolated within individual containers, so if I'm not mistaken there should be no conflict.